config/environments/development.rb

```diff
@@ -38,4 +38,10 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+  config.middleware.use Rack::Cors do
+    allow do
+      origins '*'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.